sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -1374,12 +1367,46 @@ public void testStringAnyInSubquery()
     );
   }
 
+  @Test
+  public void testOffHeapEarliestGroupBy()
```

Comment:
this seems already covered by other tests that removed 'skipVectorize' statements?